sw/cheri/revocation_test/boot.S

```diff
@@ -0,0 +1,51 @@
+.include "assembly-helpers.s"
+
+	.section .text, "ax", @progbits
+.zero 0x80
+	.globl start
+	.p2align 2
+    .type start,@function
+start:
+	// ca0 (first argument) contains the read-write root
+	cspecialr        ca0, mtdc
+
+	// Zero the tag memory
+	//li               a1, 0x200fe000
+	//csetaddr         ca0, ca0, a1
+	//li               a1, 0x20100000
+	//cjal             zero_memory
```

Comment:
Could these commented out sections be removed?

Reply:
I think so, this is copied from the CHERI sanity test and I left it in for reference. Let's do a more general cleanup in the software in a later PR.